returnn/__main__.py

```diff
@@ -231,6 +231,11 @@ def load_data(config, cache_byte_size, files_config_key, **kwargs):
         kwargs["cache_byte_size"] = cache_byte_size
     Dataset.kwargs_update_from_config(config, kwargs)
     data = init_dataset(kwargs)
+  elif config.is_typed(files_config_key) and isinstance(config.typed_value(files_config_key), Dataset):
+    # Config contains dataset object, load it
+    data = config.typed_value(files_config_key)
+    data.initialize()
+
```

Comment:
Why do you add the empty line? The code should be consistent. Either empty line before all `if`/`elif`, or none.